Framework/PythonInterface/plugins/algorithms/MaskBTP.py

```diff
@@ -172,7 +172,10 @@ def _getEightPackHandle(self,banknum):
             else:
                 raise ValueError("Out of range index for ARCS instrument bank numbers")
         elif self.instname=="SEQUOIA":
-            if self.bankmin[self.instname]<=banknum<= 74:
+            # there are only banks 23-26 in A row
+            if self.bankmin[self.instname]<=banknum<= 23+4-1:
+                return self.instrument.getComponentByName("A row")[banknum-23][0]
```

Comment:
There should be a test here that the instrument has row A. Older files fail on this. 
```
Error in execution of algorithm MaskBTP:
'NoneType' object is not subscriptable
  at line 111 in '/home/3y9/Mantid/mantid/Framework/PythonInterface/plugins/algorithms/MaskBTP.py'
  caused by line 177 in '/home/3y9/Mantid/mantid/Framework/PythonInterface/plugins/algorithms/MaskBTP.py'
```